Dockerfile

```diff
@@ -0,0 +1,68 @@
+# This is the layer that can run things
+FROM debian:buster
+LABEL maintainer="Binary.com <binary@cpan.org>"
+
+# Some standard server-like config used everywhere
+ENV TZ=UTC
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Use an apt-cacher-ng or similar proxy when available during builds
+ARG DEBIAN_PROXY
+
+WORKDIR /usr/src/perl
+
+RUN [ -n "$DEBIAN_PROXY" ] \
+ && (echo "Acquire::http::Proxy \"http://$DEBIAN_PROXY\";" > /etc/apt/apt.conf.d/30proxy) \
+ && (echo "Acquire::http::Proxy::ppa.launchpad.net DIRECT;" >> /etc/apt/apt.conf.d/30proxy) \
+ || echo "No local Debian proxy configured" \
+ && apt-get update \
+ && apt-get dist-upgrade -y -q --no-install-recommends \
+ && apt-get install -y -q --no-install-recommends \
+    git openssh-client curl socat ca-certificates gcc make libc6-dev libssl-dev zlib1g-dev xz-utils dumb-init \
+ && curl -SL https://www.cpan.org/src/5.0/perl-5.32.0.tar.xz -o perl-5.32.0.tar.xz \
```

Comment:
Could you extract the Perl version (and sha256sum) to env vars, to make future updates easier?